### PR TITLE
fix: increased the  zIndex value

### PR DESCRIPTION
**Pull request by jayanand05**
Closes: #6021 

### Description:
z-index issue. When the dropdown was open and I scroll down the page, it remained in front of the navbar, increase the zIndex value to fix it.

https://github.com/lightdash/lightdash/assets/113882904/daf6e0dd-769a-47d6-a098-4b20abab2667

### CODE REVIEW

File: packages/frontend/src/components/NavBar/index.tsx [hunk 1]
```diff
@@ -72,7 +72,7 @@ const NavBar = memo(() => {
                 display="flex"
                 px="md"
                 // FIXME: adjust after removing Blueprint
-                zIndex={10}
+                zIndex={999}
```
Comment (joaoviana) [inline packages/frontend/src/components/NavBar/index.tsx:75]: I've just checked the `zIndex` applied to the `Select` dropdown and it's `300`. Could we go for `500` here? What do you think? 😃

**Review by joaoviana — CHANGES_REQUESTED** [review]
Thanks for contributing @jayanand05 🙌 

Could you remove the changes to the `yarn.lock` file? 👍

File: packages/frontend/src/components/NavBar/index.tsx (hunk 1 above)
Comment (jayanand05) [reply packages/frontend/src/components/NavBar/index.tsx:75]: I checked it's working fine with 300 as well we can go for 500 but 300 is also that we can stick with.

Comment (joaoviana) [reply packages/frontend/src/components/NavBar/index.tsx:75]: let's go with `300` then pls @jayanand05 👍

Comment (jayanand05) [reply packages/frontend/src/components/NavBar/index.tsx:75]: 👍👍

**Comment by joaoviana** [comment]
One final thing @jayanand05, could you re-add the `yarn.lock` please? 
Ensure you are on node 16, and then run `yarn`. The file should be added again. 
Or you can remove the commits that made that change? Thanks!

File: packages/frontend/src/components/NavBar/index.tsx (hunk 1 above)
Comment (jayanand05) [reply packages/frontend/src/components/NavBar/index.tsx]: I Updated necessary changes request you to look into it?

**Comment by joaoviana** [comment]
@all-contributors add <github-handle> for code

> can you tell me where to re-add yarn.lock file ?

sure! 
so on the file changes for this PR, you can see that `yarn.lock` was deleted on the root folder: 
image

You can bring it back in 2 ways: 
* delete commits (with rebase) that changed that file 
or 
* re-run `yarn` in root: `/`

**Comment by jayanand05** [comment]
Hi @joaoviana I ran the yarn in root how to check re-adding was successful?

**Comment by joaoviana** [comment]
> Hi @joaoviana I ran the yarn in root how to check re-adding was successful?

try to commit that `yarn.lock` file and push to your branch. Check this PR - the files changed - and you should see only 1 change: the increase of the `zIndex`!

**Comment by joaoviana** [comment]
@jayanand05 `zIndex` is still `999`, could you change to `300`? 
also, there's an empty line in `yarn.lock`, maybe remove it? and push it. Then you should end up with just one file change 👍

**Comment by owlas** [comment]
@all-contributors add @jayanand05 for code

**Comment by owlas** [comment]
duplicated and fixed on: https://github.com/lightdash/lightdash/pull/6049

**Pull request closed without merging**